### ISSUE TITLE
RFC: should this repo exist?

### ISSUE THREAD

**Issue by sndrs** [guardian/client-side-infra#1]
Following [a suggestion in chat](https://chat.google.com/room/AAAAEOkegfM/7rQO_JdMO7s) I've put this repo together so we can use RFC issues to try and structure conversations in a more followable and discrete way.

What do people think? A good idea? more noise? Happy to close it down if its not useful.

**Comment by SiAdcock** [comment]
On the one hand, I think it's really useful to have a more permanent record of decisions and how we reached them. Writing comments in here requires a bit of thought: structuring arguments requires a deeper exploration of the problem space, digging up links to articles requires extra time spent on research. It's a useful forcing function to help us make better decisions.

On the other hand, I also enjoy the quickfire, seat-of-the-pants, ephemeral chats we have in `Digital/Client Side Infra`: a safe space to throw things at the wall to see what sticks, without a long memory.

In short, I'm happy that both channels exist 😄

**Comment by oliverlloyd** [comment]
> Writing comments in here requires a bit of thought: structuring arguments requires a deeper exploration of the problem space,

This is a really good point. There's a lot to be said to having a place for a more cerebral, slower discussion. So I say yes, it _should_ exist.

I also think there's no reason we can't do seat-of-the-pants silly here. I have no problem with my gif choices being documented for eternity.

**Comment by sndrs** [comment]
> Writing comments in here requires a bit of thought

i like this aspect a lot. it feels to me a bit like an asynchronous f2f meeting (in a good way).

things/input can be missed in the quickfire back and forth of chat, esp if you don't have a lot of bandwidth.

issues here are likely to have specific subjects that require deeper thought, and chat is maybe where _which_ subjects we end up discussing here will arise?

gifs can accompany everything

**Comment by AWare** [comment]
I agree that the synchronous and highly dimensional* nature of chat is also a poor choice for this and think this is much better.

*many rooms having many threads with more than one conversation in each thread

**Comment by gtrufitt** [comment]
I am 100% for async communication so this is a +1 for me. 

![tenor-248008986](https://user-images.githubusercontent.com/638051/83860006-9fc79e00-a716-11ea-93d6-cc39ef0e9f32.gif)

I'm not sure, as maybe chat meets the purpose, but https://help.github.com/en/github/building-a-strong-community/about-team-discussions might have a place somewhere.

**Comment by sndrs** [comment]
ooooooh interesting. maybe that's a better option than rfcs in a repo?

**Comment by rupertbates** [comment]
I agree that team discussions looks interesting, it might be a more natural fit for our purposes than a repo which after all is primarily designed for managing code.

I'd also suggest that we consider adopting [Architectural Decision Records](https://adr.github.io/) to capture, finalise and record the decisions that we come to (via whatever mechanism)

**Comment by oliverlloyd** [comment]
> I'd also suggest that we consider adopting Architectural Decision Records 

Very much agree! Where do you think the best place for these to live would be?
